rerun_notebook/src/js/widget.ts

```diff
@@ -87,6 +87,8 @@ class ViewerWidget {
       canvas.style.minHeight = "";
       canvas.style.maxHeight = "";
     }
+
+    canvas.tabIndex = 0;
```

Comment:
This should be done in `web-viewer` when we construct the canvas

Reply:
I actually do it in `eframe` now: https://github.com/emilk/egui/pull/4718/commits/4147c94fd7fcaa5998965a991b4996ea621fe144